setup/ssl.sh

```diff
@@ -0,0 +1,94 @@
+ssl() {
```

Comment:
Please add a condition to ensure that this feature is only run if running on a unix system